spec/support/streaming_server_manager.rb

```diff
@@ -109,6 +109,9 @@ def stop
       # Also needs to be set per-example here because of the database cleaner.
       Setting.registrations_mode = 'open'
 
+      # Load seeds so we have the default roles otherwise cleared by `DatabaseCleaner`
+      Rails.application.load_seed
```

Comment:
That is pretty awkward and I lost a lot of time on this. Not sure if it's the best way to ensure the seed is loaded.